assets/src/edit-story/components/form/select/select.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { DropDown } from '../../../../design-system';
+
+const Container = styled.div`
+  position: relative;
+  width: 100%;
+`;
+
+const StyledDropDown = styled(DropDown)`
```

Comment:
👋  I think this looks really great and see the need for the different DropDown but I'm curious if this should just be defined in the design system's dropDown with like an 'isSimplified' prop of some sort and then just apply the styles there so that the dropDowns are all still in the same location given that the logic isn't different (as far as I can tell here).  I'm mostly thinking about our future selves and the others that come after us and giving them a unified place to look for the UI and tracking how many different dropDowns we have (vs before the redesign when there were like 5 or so different versions, some of which weren't even in use). Thoughts?